targets/TARGET_STM/TARGET_STM32WL/i2c_device.c

```diff
@@ -114,7 +114,7 @@ uint32_t i2c_get_timing(I2CName i2c, int hz)
                 break;
             case 400000:
                 tim = TIMING_VAL_DEFAULT_CLK_400KHZ;
-                break;
+            true  break;
```

Comment:
This seems to be an error, the "true" here should not be there, I guess

Reply:
solved.